masterlist.yaml

```
@@ -13535,11 +13545,9 @@ plugins:
       - Outfits.Add
       - Stats
       - Text
-    dirty:
-      - <<: *quickClean
-        crc: 0x80E5FDAC
-        util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
-        itm: 1
+    clean:
+      - crc: 0x01E10725
+        util: 'SSEEdit v4.0.3'
```

Same as above, it would be best to put it in a separate pull request together with the new patch message.
```suggestion
    dirty:
      - <<: *quickClean
        crc: 0x80E5FDAC
        util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 1
```